### ISSUE TITLE
Settings screen not accessible anymore

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#313]
When using voiceover on Settings screen:

When swiping from left to right, the focus will not go to anywhere. Earlier, the focus could be moved to next item.

**Comment by giftydeep** [comment]
I don't find any issue. The only concern is the user needs to know that the current focus is on the tab not on the screen, User needs to swipe back or tap anywhere on the screen to fast-focus on the screen

**Comment by Rajat-Sen** [comment]
Validated!
